src/nemo_safe_synthesizer/cli/wandb_setup.py

```
@@ -123,18 +130,176 @@ def log_failure_to_wandb(error: Exception, phase: str) -> None:
         if wandb.run is not None:
-            wandb.log(
+            wandb.run.summary.update(
                 {
                     "eval/success": 0,
                     f"{phase}/error_type": type(error).__name__,
                     f"{phase}/error_message": str(error),
                 }
             )
-            logger.info(f"Logged failure to wandb for {phase} phase")
+            logger.info(f"Updated wandb failure summary for {phase} phase")
     except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)